### DISCUSSION TITLE
用apt安装包总返回404，有什么可替代的源推荐吗

### DISCUSSION THREAD

**Discussion by akimoto72738** [LingmoOS/Community#10] [问题解答] [answered]
![image](https://github.com/user-attachments/assets/ef52c3da-c7c8-4f6a-b404-b805c55022a6)

**Comment by chun-awa** [comment]
运行`sudo apt update`

**Comment by elysia-best** [comment, marked as answer]
请将源更换为 `deb [trusted=yes] https://packages.lingmo.org/lingmo hydrogen main contrib non-free non-free-firmware`
